core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -873,8 +873,8 @@ public static function addPagesBreadcrumb($strKey='tl_page_node')
 
 				$arrIds[] = $intId;
 
-				// No link for the active page
-				if ($objPage->id == $intNode)
+				// No link for the active page or pages in the trail
+				if ($objPage->id == $intNode || (!$objUser->isAdmin && !$objUser->hasAccess($objPage->id, 'pagemounts')))
```

Comment:
```suggestion
				if ($objPage->id == $intNode || !$objUser->hasAccess($objPage->id, 'pagemounts'))
```

`User::hasAccess()` already checks `isAdmin`, therefore this condition can possibly be simplified.

Reply:
Right, I just copied the original code ;)